base_classes/NXdata.nxdl.xml

```diff
@@ -398,21 +398,43 @@
 	</field>
 
 	<!-- Data vs. plot coordinates -->
-	<field name="scaling_factor" type="NX_FLOAT">
+	<field name="FIELDNAME_scaling_factor" type="NX_FLOAT">
 		<doc>
-			The elements in data are usually float values really. For
-			efficiency reasons these are usually stored as integers
-			after scaling with a scale factor. This value is the scale
-			factor. It is required to get the actual physical value,
-			when necessary.
+			An optional scaling factor to apply to the values in FIELDNAME (usually the signal).
+
+			The elements in data are often stored as integers for efficiency reasons and need
+			further correction, generating floats.  The two fields FIELDNAME_scaling_factor and
+			FIELDNAME_offset allow linear corrections using the following convention:
+
+			.. code-block::
+
+				plotted_data = (data + offset) * scaling_factor
+
+			This formula will derive the value to plot, when necessary.
 		</doc>
 	</field>
-	<field name="offset" type="NX_FLOAT">
+	<field name="FIELDNAME_offset" type="NX_FLOAT">
 		<doc>
-			An optional offset to apply to the values in data.
+			An optional offset to apply to the values in FIELDNAME (usually the signal).
+
+			See :ref:`FIELDNAME_scaling_factor &lt;/NXdata/FIELDNAME_scaling_factor-field&gt;` for more information.
 		</doc>
 	</field>
 
+	<field name="scaling_factor" type="NX_FLOAT" deprecated="Use FIELDNAME_scaling_factor instead">
+		<doc>
+			Due to scaling_factor being ambiguous in the case of multiple signals, use
```

Comment:
A couple of points (both minor)

1. I suggest having some throw-away comment describing the intended semantics (e.g., "Had similar semantics to FIELDNAME_scaling_factor").  This is to allow someone reading the spec to understand how to interpret existing data, where `scaling_factor` has already been used.

    This comment also applies to the `offset` field.

2. Since the ambiguity comes in the case where multiple signals are present (per the proposed doc), I'm assuming the ambiguity doesn't exist if there is a single signal.  For single signal data, is `scaling_factor` still deprecated?  I would imagine so, but perhaps the wording could be made more explicit; for example, by making the statement "use FIELDNAME_scaling_factor instead" a separate sentence, perhaps qualifying it by saying something like "all future data should use ..." . 

    This comment also applies to the `offset` field.

I consider neither comment blocking